### DISCUSSION TITLE
how can I change the language of the tags and categories ?

### DISCUSSION THREAD

**Discussion by iskerwin** [adityatelange/hugo-PaperMod#672] [Q&A]
Hello,  how can I change the language of the tags and categories ?

**Comment by Thorgrimson** [comment]
To change the language one could use defaultContentLanguage: "de" (in this example for german) in the config.yml
But for categories and tags you have to change the taxonomies in config.yaml and/or configTaxo.yml
```
taxonomies:
   tag: schlagworte
```
But for that you have to change the frontmatter in the content-files, too
```
----
schlagworte: ["tag1","tag2"]
---
```

**Reply by iskerwin** [reply]
THX
